web/styles/app_components.css

```diff
@@ -149,7 +149,7 @@ input::placeholder {
         }
 
         &.small {
-            font-size: 0.8rem;
+            font-size: calc(16 / 14 * 0.8 * var(--base-font-size-px));
```

Comment:
This always results in a pixel value, and I don't understand why this complicated `calc()` function is needed here. Let's just express this as ems:

First,

0.8rem = 16px * 0.8 = 12.8px

That's the value shown at the legacy 14px value (since rem is always based on the browser's 16px default), so let's get that legacy-derived em value:

12.8px / 14px = 0.9143em

That's 12.8px at 14px, and 14.6288px at 16px. So, `font-size: 0.9143em` is the ticket here.

Reply:
Also, make sure you note the origin of the em value in a comment:

```css
/* 12.8px at 14px/1em */
```

Reply:
From the PR description:

> The original plan was to use em instead of rem and multiply it by (16/14 * 0.8), but since em depends on the parent element, there was a case in the poll button widget where 1em was equal to 16px in the dense 14px mode. While, the right thing to do might have been some refactor to make it work as desired, the safest thing to do right now might be to use the --base-font-size-px variable directly for the 9.0 release.

The issue faced with `em` is that the `em` value will depend on the parent element font-size. 

In case of modal content with class `.modal__content`, we set the font-size to `1.1429em` which is equal to 16px when using the legacy dense mode. This means that when using the poll create button, the `em` value for the trash icon will be 16px, when we want that value to be 14px. A working `rem` unit would be really great to have, but until that happens, I don't have confidence of `em` working for `.small` font-size with changing just that one line, since the `em` value can be different for different components for all `.small` occurences.

A way to always have the same `em` value for `.small` button would be to set the font-size in `.new-style` selector, but in that case, we will end up actually changing the font-size for a lot of a content. This is because we `.new-style` with divs that might have other stuff inside which may not like it's font-size being determined by `.new-style` class.

Reply:
We get font-size as 14.63px instead of 12.8px when using 0.9143em, which also leads to the buttons becoming a bit more taller than the row accompanying them


<img width="1330" alt="Screenshot 2024-07-17 at 3 02 31 PM" src="https://github.com/user-attachments/assets/84a208fb-8708-4438-8644-cc948778349a">

Reply:
I see. `16/14*0.8` is constant, and actually *is* the `0.9143` em value, so let's maybe use that directly along with the `--base-font-size-px`, note that the result will be pixels and that this should be refactored?